docs/advanced-usage/user-quotas/index.md

```diff
@@ -0,0 +1,134 @@
+the disk or CPU usage in a human-readable format:
+
+```console
+$ reana-client quota-show --resource cpu --report limit -h
+1h
+$ reana-client quota-show --resource cpu --report usage -h
+1m 19s
+$ reana-client quota-show --resource cpu -h
+1m 19s out of 1h used (2%)
+```
+
+The usage of the resources is also displayed on the profile page of the web UI,
+which you can access by clicking on the person-shaped icon in the top-right
+corner of the web interface:
+
+!["Your quota" section of the profile page](/images/ui-profile-your-quota.png)
+
+!!! note
+    If the _"Your quota"_ section is not present, it means that user quotas
+    are not enabled in your cluster!
+
+## What to do when quotas are exceeded
+
+If you exceed your user quota of one of the resources, you will not be able to
+execute any new workflow until the resource usage will be back under the limit.
+In particular, if you have exceeded your CPU quota, there is nothing you can do
+except asking the administrators of your REANA instance to increase your CPU
+quota limit. However, if you have consumed all the available disk space, you can
+free up some of it by either deleting some past workflows or by removing some of
+their files.
+
+### Find workflows consuming the most disk space
+
+The easiest way to find out which workflows are using the most disk space is
+through the web interface. In particular, go to REANA's homepage and select the
+_"Most disk quota used"_ option in the _"Sort by"_ dropdown menu:
+
+![Sort workflows by "Most disk quota used"](/images/ui-sort-by-most-disk-quota-used.png)
+
+Workflows will then appear in decreasing order of disk usage: the heaviest
+workflows will be at the top of the list.
+
+Otherwise, you can do the same with `reana-client` and the `sort` utility:
+
+```console
+$ reana-client list --include-workspace-size | sort -n -r -k 7
+reana-demo-root6-roofit   2            2022-05-30T12:50:46   2022-05-30T12:50:58   2022-05-30T12:51:13   finished   180224
+reana-demo-helloworld     1            2022-05-30T11:59:48   2022-05-30T12:00:00   2022-05-30T12:00:15   finished   12288
+reana-demo-root6-roofit   1            2022-05-30T11:59:58   2022-05-30T12:00:09   2022-05-30T12:01:13   deleted    0
+NAME                      RUN_NUMBER   CREATED               STARTED               ENDED                 STATUS     SIZE
+```
+
+### Delete workflows
+
+If there are workflows that you do not need anymore, you can delete them to free
+up some disk space. To do so, find the workflow you want to delete, click on the
+three dots on the right to open the dropwdown menu, and select _"Delete
+workflow"_:
+
+!["Delete workflow" menu option](/images/ui-delete-workflow.png)
+
+Make sure that _"Delete also workflow workspace"_ is checked before confirming
+the deletion of the workflow, otherwise the workspace will be kept intact and
+you will not free up any space:
+
+![Dialog to confirm the deletion of a workflow](/images/ui-delete-workflow-confirmation.png)
+
```

Comment:
In the docs, you have usually provided both web UI and CLI options but, here, for deletion of workflows, it is missing. Maybe a short paragraph similar to this:

You can also delete a workflow with workspace using `reana-client`:

```
$ reana-client delete -w reana-demo-root6-roofit --include-workspace
```

For more details check `reana-client delete --help`